wavefront-adapter-for-istio/README.md

```diff
@@ -9,6 +9,26 @@ This chart will deploy the [Wavefront by VMware Adapter for Istio](https://githu
 
 ## Installation
 
+**Prerequisites**
+
+To deploy this adapter, you will need a cluster with the following setup.
+
+* Kubernetes v1.15.0
```

Comment:
Not everyone uses this specific version of Kubernetes. Is this a requirement for Istio itself or this is the only version we have tested against?

Reply:
We tested with v1.15 and v1.16 but Istio 1.4 doesn't support k8s v1.15 so just mentioned the version number which is common across all the Istio versions.